File: Semana 1/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   int change;
    do{
    change = get_int("Input the amount of money you will change:" );
    }
    while (change<0);
    int coins = 0;
    while (change>=25){
            change = change - 25;
            coins++;
        }
        while (change>=10){
            change = change - 10;
            coins++;
        }
        while (change>=5){
            change = change-5;
            coins++;
        }
        while (change>=1){
            change = change - 1;
            coins++;
        }
    printf("The amount of coins is: %i\n", coins);
    }

File: Semana 6/cash.py
from cs50 import get_float
money = get_float("Insert the amount of money: ")
while money<=0:
    money = get_float("Insert the amount of money: ")
money *= 100
coins = 0
def reduce_money(cash, amount):
    global coins
    global money
    while cash>=amount:
        cash = cash-amount
        money = cash
        coins = coins+1
reduce_money(money, 25)
reduce_money (money, 10)
reduce_money (money,5)
reduce_money (money,1)
print(coins)




